packages/grpc-native-core/templates/package.json.template

```diff
@@ -70,13 +70,15 @@
       "deps/grpc/include/grpc/**/*.h",
       "deps/grpc/src/core/**/*.{c,cc,h}",
       "deps/grpc/src/boringssl/err_data.c",
-      "deps/grpc/third_party/nanopb/*.{c,cc,h}",
-      "deps/grpc/third_party/zlib/**/*.{c,cc,h}",
+      "deps/grpc/third_party/abseil-cpp/absl/**/*.{h,hh}",
       "deps/grpc/third_party/boringssl/crypto/**/*.{c,cc,h}",
       "deps/grpc/third_party/boringssl/include/**/*.{c,cc,h}",
       "deps/grpc/third_party/boringssl/ssl/**/*.{c,cc,h}",
       "deps/grpc/third_party/boringssl/third_party/**/*.{c,h}",
-      "deps/grpc/third_party/abseil-cpp/absl/**/*.{h,hh}",
+      "deps/grpc/third_party/nanopb/*.{c,cc,h}",
+      "deps/grpc/third_party/upb/generated_for_cmake/google/protobuf/*.{c,h}",
+      "deps/grpc/third_party/upb/upb/*.{c,h}",
```

Comment:
These two lines should probably just be "deps/grpc/third_party/upb/*.{c,h}".

Reply:
Done! Thanks :)